pkg/resmap/resmap.go

```diff
@@ -93,6 +93,13 @@ type ResMap interface {
 	// Same as GetByOriginalId.
 	GetById(resid.ResId) (*resource.Resource, error)
 
+	// GroupedByNamespace provides map of discardable slice
+	// of resource pointer
+	// The not namespaceable resources are added to the "cluster-wide" key.
```

Comment:
This "cluster-wide" key value needs to be a public package constant, for clients to use.  Recommend an impossible name, e.g. "%_no_namespace_%" (since % is forbidden in names afaict)

Alternatively, don't return this particular resource list at all (in your application, you filter them out), or return it via a distinct method.